packages/hooks/README.md

```diff
@@ -1,34 +1,28 @@
 # WP-JS-Hooks
 
-A lightweight & efficient EventManager for JavaScript in WordPress.
-
+A lightweight & efficient event manager.
 
 ### API Usage
-API functions can be called via the global `wp.hooks` like this `wp.hooks.addAction()`, etc.
-
-* `addAction( 'hook', 'vendor/plugin/function', callback, priority )`
-* `addFilter( 'hook', 'vendor/plugin/function', callback, priority )`
-* `removeAction( 'hook', 'vendor/plugin/function' )`
-* `removeFilter( 'hook',  'vendor/plugin/function' )`
-* `removeAllActions( 'hook' )`
-* `removeAllFilters( 'hook' )`
-* `doAction( 'hook', arg1, arg2, moreArgs, finalArg )`
-* `applyFilters( 'hook', content, arg1, arg2, moreArgs, finalArg )`
-* `doingAction( 'hook' )`
-* `doingFilter( 'hook' )`
-* `didAction( 'hook' )`
-* `didFilter( 'hook' )`
-* `hasAction( 'hook' )`
-* `hasFilter( 'hook' )`
-
+Hooks can be added to an object via composition:
+
+`myObject.hooks = new Hooks()`
```

Comment:
Also, `Hooks` could be used as a mixin:

```js
Object.assign( myObject, new Hooks() );
```

Usage:

```js
myObject.addFilter( … );
```

Reply:
Right, I'll document and test this as well